python/shotgun_model/util.py

```diff
@@ -148,6 +148,9 @@ def sanitize_qt(val):
             new_val[safe_key] = safe_val
         return new_val
 
+    elif isinstance(val, long):
```

Comment:
we need to add some comments here to explain why we have to do this! If possible please include info about versions of pyside, QT etc. This will help maintain the code in the future and help people debug in different DCCs when issues arise! Thanks!! :)

Reply:
Hi @manneohrstrom I added some comments

Reply:
thanks great! Sorry for being nitpicky, could you tweak it a bit:
- Remember this is a public repo, everyone can see, so our jira ticket isn't really helpful for people (they cannot access our bug tracker!). I suggest just removing that. People curious about it can always follow the PR trail and find the ticket id that way.
- Please change it into a `# comment` rather than a string - that's how we normally do things!

Thanks!!!!